GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -277,6 +280,8 @@ async Task LoadLinksForRevisionAsync(GitRevision revision)
                     }
 
                     await this.SwitchToMainThreadAsync(cancellationToken);
+                    cancellationToken.ThrowIfCancellationRequested();
```

Comment:
📝 This is handled already by the extension method on the previous line.

```suggestion
```

Reply:
Thank you